src/components/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECK_STATUS = 'CHECKING STATUS';
```

Comment:
- [ ] As I mentioned in the previous review, I think it would be better if the value of each action type follows the [duck's pattern](https://github.com/erikras/ducks-modular-redux) for the project's requirements. So it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`